### DISCUSSION TITLE
Choice of editor

### DISCUSSION THREAD

**Discussion by tony-teng-yt377** [Part-II-Computational-Physics/.github#11] [General]
The animation does not work in VS for me (it shows '<IPython.core.display.HTML object>' but not the actual graph), as also encountered by other ppl. 
I put it in google colab Jupiter notebook, and it works for graph but not for animation. 

Anybody found the solution to this?

**Comment by NathanMagnan** [comment]
Hey!

If you want it to work in VS code, I suspect you need to remove the `%matplotlib notebook` at the start.
But I would prefer if everyone handed me a Jupyter notebook, if possible.

**Reply by tony-teng-yt377** [reply]
Thank you so much!

**Comment by tony-teng-yt377** [comment]
Update: put the line %matplotlib notebook at the beginning of google colab notebook, and it enables the inline display of animation!
